editor/editor_file_system.cpp

```diff
@@ -1664,7 +1664,10 @@ void EditorFileSystem::_queue_update_script_class(const String &p_path) {
 }
 
 void EditorFileSystem::_update_scene_groups() {
-	EditorProgress ep("update_scene_groups", TTR("Update Scene Groups"), update_scene_paths.size());
+	EditorProgress *ep = nullptr;
+	if (update_scene_paths.size() > 1) {
+		ep = memnew(EditorProgress("update_scene_groups", TTR("Update Scene Groups"), update_scene_paths.size()));
+	}
 	int step_count = 0;
```

Comment:
Maybe this should go in the `if` too since it's not used otherwise?

Reply:
Variable declaration can't be ifed. (which is also why I had to use pointer to EditorProgress)
To completely avoid it I'd have to repeat the whole code twice in different branches.